mesheryctl/internal/cli/root/pattern/apply.go

```diff
@@ -84,16 +83,12 @@ mesheryctl pattern apply [pattern-name]
 				return errors.Wrap(err, "could not create request ")
 			}
 
-			resp, err := client.Do(req)
+			resp, err := utils.MakeRequest(req)
 			if err != nil {
-				return errors.Errorf("unable to reach Meshery Server at %s. Verify your environment's readiness for a Meshery deployment by running `mesheryctl system check`", mctlCfg.GetBaseMesheryURL())
```

Comment:
@suhail34, I think that you're familiar with the MeshKit Errors library. Ideally, we are providing users with lots of information and have a specific error code for each failure.

@abdullah1308 @alphaX86 should we be having a MeshKit Error here?

Reply:
@leecalcote i feel having a brief error log information, will help us track the errors in the codebase incase of some future bugs.
 Maybe the error logs wouldn't be such that it would scare the users

Reply:
@suhail34 @abdullah1308 ?